packages/webpack/src/config/base.js

```diff
@@ -195,7 +195,7 @@ export default class WebpackBaseConfig {
         })
       },
       {
-        test: /\.css$/,
+        test: /\.(post)?css$/,
```

Comment:
How do you think add a separate rule for `.postcss` in case of that it may have special loader in future ?

Reply:
Sure, we can do that as well ☺️